docs/gds-supported-platform/tls-certificates.md

```diff
@@ -27,6 +32,75 @@ spec:
 
 > **Note:** cert-manager will need to be able to modify the DNS of the domains listed in the certificate in order to perform the DNS challenge. At the time of writing that only applies to the cluster domain.
 
+## Providing your own certificates
+
+If you want to provide your own certificate, you will need:
+
+ - a `SealedSecret` resource containing the certificate and key
+ - a `Gateway` resource defining the ingress domain
+
+We assume:
+
+ - you are in a namespace called `my-namespace`
+ - your certificate and key are in files called `my-cert.pem` and
+   `my-cert.key`
+ - you want to create a `SealedSecret` called `my-custom-cert` in a
```

Comment:
Line 39 states that you will need a `SealedSecret` resource containing the certificate and key. Yet this section assumes that "you want to create a `SealedSecret` called `my-custom-cert`". At this stage, do you have the `SealedSecret` or not?

Reply:
ah. line 39 means "to achieve this task, you will need to create a SealedSecret and a Gateway" and the rest of the section explains how to go about creating these things.  Maybe it could be worded better because I can see how it's confusing.